app/src/libs/objectives.ts

```diff
@@ -83,6 +83,27 @@ export const getObjectiveImage = (objective: AllObjectivesType) => {
         image: IMG_BADGE_MEDICAL_EXPERIENCE,
         title: "Medical Experience Gained",
       };
+    case "crafting_experience":
+      return { image: IMG_BADGE_MEDICAL_EXPERIENCE, title: "Crafting Experience" };
+    case "crafting_experience_gained":
+      return {
+        image: IMG_BADGE_MEDICAL_EXPERIENCE,
+        title: "Crafting Experience Gained",
+      };
+    case "hunting_experience":
+      return { image: IMG_BADGE_MEDICAL_EXPERIENCE, title: "Hunting Experience" };
+    case "hunting_experience_gained":
+      return {
+        image: IMG_BADGE_MEDICAL_EXPERIENCE,
+        title: "Hunting Experience Gained",
+      };
+    case "gathering_experience":
+      return { image: IMG_BADGE_MEDICAL_EXPERIENCE, title: "Gathering Experience" };
+    case "gathering_experience_gained":
+      return {
+        image: IMG_BADGE_MEDICAL_EXPERIENCE,
```

Comment:
Need icons for the new trackers. Ideally also icons compressed to e.g. 200x200 .webp format for transfer speeds